packages/gatsby-plugin-vercel-builder/src/helpers/static.ts

```diff
@@ -10,12 +11,12 @@ export async function createStaticDir(prefix?: string) {
     'static',
     prefix ?? ''
   );
-  await ensureDir(dirname(targetDir));
+
   try {
```

Comment:
We don't need the `copy()` fallback anymore since `hardLinkDir()` has that fallback itself.

Reply:
Good point! [`67a0cbb` (#9673)](https://github.com/vercel/vercel/pull/9673/commits/67a0cbbdf3aee94e778e19a8677cf39cf7b86e64)